apps/web-portal/lib/use-cases/log/LogSave.mjs

```diff
@@ -0,0 +1,9 @@
+import {UseCase} from '@zorko-io/util-use-case'
+
+export class LogSave extends UseCase {
+  // eslint-disable-next-line no-unused-vars
+  async run(logs) {
+    console.log('logs ', logs)
```

Comment:
add todo to wire with 'log' Pino instance

Reply:
done